monitor_packages/postgresql/postgresql.json

```diff
@@ -0,0 +1,322 @@
+{
+  "name": "PostgreSQL",
+  "description": "Folder for PostgreSQL Monitors",
+  "type": "MonitorsLibraryFolderExport",
+  "children": [
+    {
+      "name": "Postgresql- High Rate Deadlock",
+      "description": "This alert fires when we detect deadlocks in a Postgres instance",
+      "type": "MonitorsLibraryMonitorExport",
+      "monitorType": "Metrics",
+      "queries": [
+        {
+          "rowId": "A",
+          "query": "$$postgresql_data_source metric=postgresql_deadlocks db_system=postgresql db_cluster=* db=* host=* | quantize using sum| sum by  db_cluster, host, db | rate | eval(_value*60)"
```

Comment:
As you have custom source category, remember to mention in the readme to find and replace.

Reply:
done in docs